io.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.gnome.Platform
```

Comment:
do you really need the GNOME runtime?

Reply:
I guess. The application requires GTK and I thought GTK is available only in the GNOME runtime.

Reply:
gtk3 is in the freedesktop runtime.

Reply:
Ok, thanks for the information. Updated in 07972ec2cd2e5c0f454959fa854df2328bf5baf3.

Reply:
Runtime changed in 07972ec2cd2e5c0f454959fa854df2328bf5baf3.